.circleci/config.yml

```diff
@@ -155,6 +155,27 @@ jobs:
             just simulate-council
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
+  just_simulate_009-fp-upgrade:
```

Comment:
Need to add this job to the bottom too so it actually runs, it's currently unused

Reply:
every time.
fixed